map_model/src/make/transit.rs

```diff
@@ -238,6 +247,15 @@ fn create_route(
                 req
             );
         }
+        if req.start.lane().road == req.end.lane().road
+            && req.start.dist_along() > req.end.dist_along()
+        {
+            bail!(
```

Comment:
I saw this near a big bus station. Maybe the stops in GTFS were out of order, maybe there was some bug -- for now, just give up on the route